cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-admin.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-admin"]
+= Tutorial: Creating an admin user
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-admin
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-27
+
+Creating an administration (admin) user allows you to access your cluster quickly. Follow these steps to create an admin user. 
+
+[NOTE]
+====
+An admin user works well in this tutorial setting. For actual deployment, use a formal identity provider to access the cluster and grant the user admin privileges.
+====
+
+. Run the following command to create the admin user:
++
+[source,terminal]
+----
+rosa create admin --cluster=<cluster-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
+I: Admin account has been added to cluster 'my-rosa-cluster'. It may take up to a minute for the account to become active.
+I: To login, run the following command:
+oc login https://api.my-rosa-cluster.abcd.p1.openshiftapps.com:6443 \
+--username cluster-admin \
+--password FWGYL-2mkJI-00000-00000
+----
+
+. Copy the log in command returned to you in the previous step and paste it into your terminal. This will log you in to the cluster using the CLI so you can start using the cluster.
++
+[source,terminal]
+----
+$ oc login https://api.my-rosa-cluster.abcd.p1.openshiftapps.com:6443 \
```

Comment:
Could we/should we use the ROSA CLI in these examples vs the OC CLI? Not sure, but I would think it would make sense to use the ROSA CLI? Maybe not... :(

Reply:
I think that I will leave it as `oc` here since that is what QE approved.